lib/naughty/core.lua

```diff
@@ -428,8 +428,11 @@ end
 -- @tparam[opt] func args.callback Function that will be called with all arguments.
 --   The notification will only be displayed if the function returns true.
 --   Note: this function is only relevant to notifications sent via dbus.
--- @tparam[opt] table args.actions Mapping that maps a string to a callback when this
---   action is selected.
+-- @tparam[opt] table args.actions Table describing all of the actions that are
+--   supported by the notification client. The table key is an action identifier.
+--   The table value is a record table indexed by two keys: `text` is a localized
+--   description string which is displayed to a user, and `callback` is a function
+--   called when this action is selected.
```

Comment:
Why do you have to change this? Are you worried that two actions will have the same text?
If not, then the old `text = callback` is just as expressive as `{ text = "foo", callback = callback }` (although it may be more future proof and extendable...

I'm not really sure about what is better. Feel free to leave it like this if no one else complains.